frontend/src/scenes/insights/Insights.tsx

```diff
@@ -142,20 +187,70 @@ export function Insights() {
                         ),
                     }}
                 >
-                    <TabPane tab={<span data-attr="insight-trends-tab">Trends</span>} key={ViewType.TRENDS} />
-                    <TabPane tab={<span data-attr="insight-funnels-tab">Funnels</span>} key={ViewType.FUNNELS} />
-                    <TabPane tab={<span data-attr="insight-sessions-tab">Sessions</span>} key={ViewType.SESSIONS} />
-                    <TabPane tab={<span data-attr="insight-retention-tab">Retention</span>} key={ViewType.RETENTION} />
-                    <TabPane tab={<span data-attr="insight-path-tab">User Paths</span>} key={ViewType.PATHS} />
+                    <TabPane
+                        tab={
+                            <span data-attr="insight-trends-tab">
+                                Trends
+                                <InsightHotkey hotkey="t" />
+                            </span>
+                        }
+                        key={ViewType.TRENDS}
+                    />
```

Comment:
These things look pretty dominant to me.

![image](https://user-images.githubusercontent.com/53387/112504049-b5807f80-8d8b-11eb-9831-a3b8d062be92.png)

1. They shouldn't really show up when you don't have a keyboard (so tablet or mobile). 
2. Even on a computer, they take up quite a bit of space and I don't think I want to be constantly reminded of hotkeys.  What about just underlining the right letters like in this perfect interface:
![image](https://user-images.githubusercontent.com/53387/112504240-e3fe5a80-8d8b-11eb-9f27-94401091eb1e.png)

Reply:
Agreed on hiding this on devices without keyboards, will look for the best supported option (i.e. consider iPads with Keyboards, MS Surface, ...). I think this more obvious approach could help discoverability at first, but I also see your point. Let's A/B test this!

Reply:
If it's behind a feature flag, then fine to merge like this. This is a very subjective visual change though, so I'd ask for a lot of opinions...

Reply:
Also, I don't think there's any way to accurately detect if there's a keyboard or not. You can ignore broad category (all "Android" and "iOS"), but you can't know this from any API as far as I know.

Reply:
Definitely! Will open an issue to get opinions on this and/or A/B test for a subsequent improvement. Yeah, there are some hacks but nothing that covers all edge cases well. I think for now I'll exclude mobile devices altogether.